auth-backend/src/middleware/auth.middleware.ts

```
@@ -0,0 +1,50 @@
+import { validateAccessToken, validateRefreshToken } from '../utils/helpers';
+import {
+  generateJWTToken,
+  saveRefreshTokenToRedis,
+} from '../token/jwt-token-manager';
+import { encryptData } from '../encryption';
+import { setCookies } from '../handlers/user-handler';
+
+
+export const validateAuthMiddleware = async (req: Request, res: Response) => {
+  const [access, refresh] = req!.headers!.authorization!.split(' ,');
+  const accessToken = access && access.split('=')[1];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)